hail_scripts/shared/elasticsearch_client.py

```diff
@@ -1,369 +0,0 @@
-import datetime
-import inspect
-import logging
-import time
-from pprint import pformat
-
-try:
-    import elasticsearch
-except ImportError:
-    import os
-    os.system("pip install elasticsearch")
-    import elasticsearch
-
-
-handlers = set(logging.root.handlers)
-logging.root.handlers = list(handlers)
-logger = logging.getLogger()
-
-
-class ElasticsearchClient:
```

Comment:
This is used in the sv pipeline and by the luigi pipeline through hail_script utils.

Reply:
so it actually isn't used as this only works for the old v6 elasticsearch that no longer exists and won't work with v7 elasticsearch which is what we have live. The sv_pipeline and hail_scripts/v02/utils/elasticsearch_client.py use the v7 elasticsearch client imported from `hail_scripts.shared.elasticsearch_client_v7` which I left in the repo, and that file does not inherit from this file at all. And the luigi piepline actually imports `hail_scripts.v02.utils.elasticsearch_client` which, as I mentioned, itself imports the v7 client. So its a REALLY confusing series of names and inheritance, but I'm really confident this isn't used

Reply:
ugh yeah, I was definitely getting confused by naming-- I was reading hail_scripts/shared/elasticsearch_utils.py, not client. 🤦